src/JuliusSweetland.OptiKey.Core/UI/ViewModels/MainViewModel.LookToScroll.cs

```diff
@@ -59,6 +59,13 @@ public Thickness ActiveLookToScrollMargins
             private set { SetProperty(ref activeLookToScrollMargins, value); }
         }
 
+        private Timestamped<double> opacity = new Timestamped<double>(1.0, DateTime.Now);
```

Comment:
@AdamRoden What's the purpose of Timestamping the opacity double? It doesn't seem to be used.

Reply:
Good point. I had something else in mind when I set it up that way. No need now.

Reply:
Do you want me to wait for this correction from you, or I am happy to remove it after merging? I will only be able to look this evening though.